### ISSUE TITLE
Util not exported on reactstrap for typescripts

### ISSUE THREAD

**Issue by pedox** [reactstrap/reactstrap#1789]
- components: `Util`
- reactstrap version `8.4.1`
- import method `es` 
- react version `16.12.0`
- bootstrap version `4.4.1`

### What is happening?

is Util is not exported on reactstrap typescript?  i see in main scripts Util exported by default
i wanted to call `Util.setGlobalCssModule` seems not work in typescript but worked in js

### What should be happening?

should exported on `import { Util } from 'reactstrap'`

### Steps to reproduce issue

1. Adding type definition on `@types/reactstrap` ?
### Error message in console

```
Module '"../../../../../../node_modules/@types/reactstrap"' has no exported member 'Util'.
```

### Code

```tsx
import React from 'react'
import { Button, Util } from 'reactstrap'
import bs from 'styles/app.module.scss'
interface Props {}

const index: React.FC<Props> = () => {
  Util.setGlobalCssModule(bs)
  return (
    <div>
      <Button>Test</Button>
      <h1>Hold the line</h1>
    </div>
  )
}

export default index
```

**Comment by kyletsang** [comment]
Looks like all of the utils functions were exported, but some (most?) of them seem to be intended for internal usage only. 

@TheSharpieOne, which of these are meant to be public facing other than `setGlobalCssModule`?

**Comment by TheSharpieOne** [comment]
They are all really for internal use, but they are exported because "why not". You already have the code in your bundle, if you want to be able to use it, you should be able to.

As for typescript definitions, those are in a separate repo.

**Comment by kyletsang** [comment]
Thanks.  I'll add the TS definition for setGlobalCssModule in the other repo.

**Comment by pedox** [comment]
@kyletsang Thank you for this, much appreciated

**Comment by kyletsang** [comment]
This is now available in 8.4.2 of `@types/reactstrap`.  Please close this issue.
